pyke/kepmask.py

```diff
@@ -160,19 +162,23 @@ def kepmask(infile, frameno=100, maskfile='mask.txt', plotfile='kepmask.png',
     xmax = xmin + xdim
 
     # intensity scale
-    pimg, imin, imax = kepplot.intScale1D(pimg, zscale)
-    if zmin and zmax and zscale=='logarithm':
-        zmin = math.log10(zmin)
-        zmax = math.log10(zmax)
-    elif zmin and zmax and zscale=='squareroot':
-        zmin = math.sqrt(zmin)
-        zmax = math.sqrt(zmax)
-    elif zmin and zmax and zscale=='linear':
-        zmin *= 1.0
-        zmax *= 1.0
+    if imin is None and imax is None:
+        imin, imax = MinMaxInterval().get_limits(pimg)
     else:
```

Comment:
Does `MinMaxInterval` produce reasonable images?  It could get thrown off by a single outlier.  I think we want to use e.g. `visualization.PercentileInterval` instead, which I think is more similar to the crazy interval determination that goes on inside `kepplot.intScale1D`.